tests/acceptance/PhpstormGenerics.feature

```diff
@@ -0,0 +1,37 @@
+Feature: Phpstorm Generic Support
```

Comment:
Unrelated to this PR at hand, but I was curious if it was working or not. I included it here 🤷